blackbox/docs/config/index.rst

```diff
@@ -69,4 +70,4 @@ settings will override configuration file settings.
    cluster
    session
    logging
-   environment
\ No newline at end of file
+   environment
```

Comment:
it looks like you added an empty newline above this. this should be taken back out

Reply:
No empty line there, I can also not recall having done so.